tck/src/main/java/org/eclipse/microprofile/rest/client/tck/sse/ReactiveStreamsPublisherTckTest.java

```diff
@@ -26,10 +23,10 @@
 import java.util.concurrent.TimeUnit;
 import java.util.concurrent.atomic.AtomicBoolean;
 import java.util.concurrent.atomic.AtomicReference;
-
 import javax.ws.rs.sse.InboundSseEvent;
-
 import org.eclipse.microprofile.rest.client.RestClientBuilder;
+import static org.eclipse.microprofile.rest.client.tck.sse.AbstractSseTest.PORT;
+import static org.eclipse.microprofile.rest.client.tck.sse.AbstractSseTest.launchServer;
```

Comment:
Would you undo the import order changes?  My guess is that your IDE did that for you automatically - but there may be a switch to disable that save-action.

Reply:
Exactly as you said. Reverted.